packages/autocomplete-core/src/utils/mapToAlgoliaResponse.ts

```diff
@@ -4,8 +4,24 @@ import type {
 } from '@algolia/client-search';
 
 export function mapToAlgoliaResponse<THit>(
-  results: Array<SearchResponse<THit> | SearchForFacetValuesResponse>
+  rawResults: Array<SearchResponse<THit> | SearchForFacetValuesResponse>
 ) {
+  const results: Array<
+    SearchResponse<THit> | SearchForFacetValuesResponse
+  > = rawResults.map((result) => {
+    return {
+      ...result,
+      hits: (result as SearchResponse<THit>).hits?.map((hit) => {
```

Comment:
I don't see a check for sffv response here, are you relying on it being undefined?

Reply:
Yes—we don't transform `hits` if it's not defined.

We don't support Insights for facet hits.